@xen-orchestra/fs/src/s3.js

```diff
@@ -1,12 +1,26 @@
-import aws from '@sullux/aws-sdk'
+import {
+  AbortMultipartUploadCommand,
+  CompleteMultipartUploadCommand,
+  CopyObjectCommand,
+  CreateMultipartUploadCommand,
+  DeleteObjectCommand,
+  GetObjectCommand,
+  HeadObjectCommand,
+  ListObjectsV2Command,
+  PutObjectCommand,
+  S3Client, UploadPartCommand,
+  UploadPartCopyCommand,
+} from '@aws-sdk/client-s3';
+import { Upload } from '@aws-sdk/lib-storage';
+import { NodeHttpHandler } from '@aws-sdk/node-http-handler';
 import assert from 'assert'
-import http from 'http'
-import https from 'https'
```

Comment:
Why this change?

Reply:
I rewrote the part of the configuration creating instances of these agents, so I imported them the same way we are importing other items. No particular reason.